src/credentials/LastKnownGoodTime.cpp

```diff
@@ -35,6 +35,7 @@ constexpr TLV::Tag kLastKnownGoodChipEpochSecondsTag = TLV::ContextTag(0);
 
 void LastKnownGoodTime::LogTime(const char * msg, System::Clock::Seconds32 chipEpochTime)
```

Comment:
This function is used only in the LastKnownGoodTime.cpp unit. Maybe it should be defined locally, so in case when CHIP_PROGRESS_LOGGING is not enabled it can be replaced with no-op? Right now call instructions will be generated anyway.

Reply:
Compiler should be able to easily strip calls to an empty function - I verified that it works in this case.